tcadmin/resources/hook.py

```diff
@@ -101,3 +101,12 @@ def to_api(self):
             "task": self.task,
             "triggerSchema": self.triggerSchema,
         }
+
+    def check_hook_validity(self):
```

Comment:
When does this function get called?

Reply:
Hey, have been trying to figure the entry-point for it. I found create_hook() and update_hook() in tcadmin/update.py. Can you please guide me?